README.md

```diff
@@ -6,33 +6,87 @@ At a high level the key thing you will contribute is a `Dockerfile`, which will
 
 Your `Dockerfile` will expose a simple HTTP server, which needs to implement 2 endpoints `/process` and `/tokenize`. We will build that `Dockerfile` and expect it to launch an HTTP server. Once that server is launched, we will make requests to it via HELM and record your results.
 
+At a high level the flow you should follow to ensure a strong submission is
+1. Pick approved LLMs and datasets from [here](https://llm-efficiency-challenge.github.io/challenge)
+2. Start with one of [sample-submissions](sample-submissions) and make sure it runs
+3. Evaluate it locally on your own A100 or 4090, if you don't have funding for either please see the [GPU funding](#gpu-funding) section for some more options
+4. Once you have something working you can make a submission on our [Discord Leaderboard](https://discord.com/channels/1124130156336922665/1124134272631054447/1151718598818156645) to see how you fare up against other competitors
+5. On the competition deadline make sure you have the final eval Dockerfile you'd like us to run in your github repo, refer to the [timeline](https://llm-efficiency-challenge.github.io/dates)
+6. If your entry makes the shortlist, we will work with you to reproduce all of your artifacts with another finetuning Dockerfile
+
 ## Contents
 
+- [Approved LLM & Dataset](#approved-llm-and-dataset)
 - [Submission](#submission)
-- [HELM](#helm)
+- [Eval locally](#eval-locally-helm)
+- [Finetune](#finetune)
+- [Create your own submission template](#your-own-submission-template)
+- [Discord Leaderboard](#discord-leaderboard)
+- [Final eval submission](#final-eval-submission)
+- [Final finetune submission](#final-finetune-submission)
+- [GPU funding](#gpu-funding)
 
-## Submission
+## Approved LLM and dataset
 
-The submission in this repository is a basic implementation of the setting up an HTTP server in accordance to the open_api spec. It includes a sample solution built off of [Lit-GPT](https://github.com/Lightning-AI/lit-gpt) and open-llama weights that participants can reference or modify as they see fit.
+The LLM space has complex licenses which can make it difficult to figure out what's permittable to use in a competition to streamline this process we've shortlisted a few models and datasets we know are safe to use [here](https://llm-efficiency-challenge.github.io/challenge)
 
-### Usage
+That said the LLM space is fast moving so if you'd like to use a dataset or model that isn't on our list make sure to ask us about it on [https://discord.gg/XJwQ5ddMK7](https://discord.gg/XJwQ5ddMK7)
+
+## Submission
+
+The submission in this repository is a basic implementation of the setting up an HTTP server in accordance to the `open_api` spec. It includes a sample solution built off of [Lit-GPT](https://github.com/Lightning-AI/lit-gpt) and open-llama weights that participants can reference or modify as they see fit.
```

Comment:
I would make sure people understand that creating one (and only one) Dockerfile is a requirement.
Also that you'll need to host your weights somewhere that will be downloaded either during docker build or at runtime (i.e. not in the repo / zip file).